### PR TITLE
Upload ShortGames

### PR DESCRIPTION
**Pull request by laHaniid**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/shortgames/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "shortgames",
+  "nice_name": "ShortGames",
+  "direct_ip": "shortgames.eu",
+  "social": {
+    "web_shop": "https://shop.shortgames.eu",
+    "discord": "https://dsc.gg/shortgames"
```
Comment (maaaathis) [inline minecraft_servers/shortgames/manifest.json:7]: Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead

File: minecraft_servers/shortgames/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "shortgames",
+  "nice_name": "ShortGames",
+  "direct_ip": "shortgames.eu",
+  "social": {
+    "web_shop": "https://shop.shortgames.eu",
```
Comment (maaaathis) [inline minecraft_servers/shortgames/manifest.json:6]: Returns a Cloudflare 1014 Error
image

File: minecraft_servers/shortgames/icon@2x.png
Comment (maaaathis) [inline minecraft_servers/shortgames/icon@2x.png:1]: Compress this file

File: minecraft_servers/shortgames/backround.png
Comment (maaaathis) [inline minecraft_servers/shortgames/backround.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/shortgames/backround.png
Comment (laHaniid) [reply minecraft_servers/shortgames/backround.png:1]: What is there wrong?

File: minecraft_servers/shortgames/background@2x.png
Comment (maaaathis) [inline minecraft_servers/shortgames/background@2x.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**